packages/core/botpress/src/server/api.js

```diff
@@ -187,11 +187,14 @@ bp.createShortlink('chat', '/lite', {
 
     app.get(`/s/:name`, (req, res) => {
       const name = req.params.name.toLowerCase()
+      const query = qs.stringify(req.query)
 
       if (!links[name]) {
         return res.status(404).send({ error: `Shortlink "${name}" not registered` })
       }
 
+      links[name] = links[name].concat(`&${query}`)
```

Comment:
Same as `links[name].push(`&${query}`)`? Also should we only do it when query is not empty? Also are we sure there's always `?` in the URL? From what I see [here](https://github.com/dmytropanontko/botpress/blob/f4824ed0b8d7dbcf1c9c12d566c7249a9e39f188/packages/core/botpress/src/server/api.js#L185) it's not true in which case having an ampersand without the `?` will make the URL invalid.

Reply:
@emirotin ty

Reply:
@emirotin how you think does it enough _check_ next `/\?/g.test_(links[name])`

Reply:
Could simply be `links[name].indexOf('?') >= 0` or `links[name].includes('?')`

Also I was wrong about `push` as I erroneously thought it's an array, not a string.
But you could use `+=` here: `links[name] += (links[name].includes('?') ? '&' : '?') + query`